server/package.json

```diff
@@ -23,6 +23,7 @@
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "knex": "^0.21.6",
+    "nodemon": "^2.0.6",
```

Comment:
Should probably change this to a dev dependency and also add a dev start script as well